source/geometries/NextDemoSiPMBoard.h

```diff
@@ -1,11 +1,12 @@
 // -----------------------------------------------------------------------------
-// nexus | NextDemoSiPMBoard.h
+// nexus | NextDemoSiPMBoard.cc
```

Comment:
Why this change? It should be `.h`.

Reply:
Rigth, I just copied the description in the .cc. Amended.